thirdeye-ui/src/app/pages/welcome-landing-page/welcome-landing-page.component.tsx

```diff
@@ -0,0 +1,172 @@
+import { useTranslation } from "react-i18next";
+import { useNavigate } from "react-router-dom";
+import { LoadingErrorStateSwitch } from "../../components/page-states/loading-error-state-switch/loading-error-state-switch.component";
+import { WelcomeStepCard } from "../../components/welcome-landing-page/welcome-step-card/welcome-step-card.component";
+import { PageContentsCardV1, PageV1 } from "../../platform/components";
+import { DimensionV1 } from "../../platform/utils";
+import { ActionStatus } from "../../rest/actions.interfaces";
+import { useGetDatasets } from "../../rest/datasets/datasets.actions";
+import {
+    getDataConfigurationCreatePath,
+    getWelcomeCreateAlert,
+} from "../../utils/routes/routes.util";
+
+export const WelcomeLandingPage: FunctionComponent = () => {
+    const navigate = useNavigate();
+    const { t } = useTranslation();
+
+    const { status, datasets, getDatasets } = useGetDatasets();
+
+    const hasDatasets = !!(datasets && datasets.length > 0);
+
+    useEffect(() => {
+        getDatasets();
+    }, []);
+
+    const getParsedMarkup = (plainText: string): JSX.Element => {
+        return plainText.split("**").reduce<JSX.Element>(
+            (sum, val, i) => (
+                <>
+                    {sum}
+                    {i % 2 === 0 ? val : <strong>{val}</strong>}
+                </>
+            ),
+            <></>
+        );
+    };
+
+    return (
+        <PageV1>
+            <Box display="flex" flexDirection="column" p={7} width="100%">
+                <PageContentsCardV1 fullHeight>
+                    <Box pt={4} px={8}>
+                        <Grid
+                            container
+                            alignItems="center"
+                            direction="row"
+                            justifyContent="center"
+                            spacing={4}
+                        >
+                            <Grid item xs={8}>
+                                <Box clone fontWeight="500" pt={2}>
+                                    <Typography variant="body1">
+                                        {t("message.welcome-to-thirdeye")}
+                                    </Typography>
+                                </Box>
+                                <Box py={2}>
+                                    <Typography color="primary" variant="h4">
+                                        {t(
+                                            "message.lets-create-your-first-setup"
+                                        )}
+                                    </Typography>
+                                </Box>
+                                <Typography variant="body1">
+                                    {getParsedMarkup(
+                                        t(
+                                            "message.by-creating-an-alert-message"
+                                        )
+                                    )}
```

Comment:
I don't know if this works but can you try out the solution in https://stackoverflow.com/questions/65975098/i18next-bold-text-in-the-middle-of-a-translation

Reply:
Ah, I wasn't aware of this component. This seems much more generic. I've implemented this. I'll read more about react-i18n.